compute_tools/src/spec.rs

```diff
@@ -697,3 +697,72 @@ pub fn handle_extension_neon(client: &mut Client) -> Result<()> {
 
     Ok(())
 }
+
+#[instrument(skip_all)]
+pub fn handle_migrations(client: &mut Client) -> Result<()> {
+    info!("handle migrations");
+    let migrations = [
```

Comment:
Here we rely on the fact that new migrations are added to the end of this array. Worth leaving a comment at least. Or better switch to array of pairs where first element is a manually-provided timestamp and value is a migration text. Then store timestamp of last migration in the database.

Reply:
What format of timestamp?